package.json

```diff
@@ -73,7 +74,10 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged"
```

Comment:
@jmcdo29 Should we remove `pretty-quick` or just leave it for now?

Reply:
`lint-staged` as we define one line 80 will run prettier. We can replace prettier with pretty-quick if you prefer, but I'm rather sure they do the same thing. It also is going to run the linter on pre-commit but that can also be modified